tools/build_third_party.py

```diff
@@ -29,3 +29,4 @@
 remove_and_symlink(join("v8", "third_party", "markupsafe"), "markupsafe")
 run(["gclient", "sync", "--shallow", "--no-history"])
 run(["yarn"])
+run(["cargo", "fetch", "--manifest-path=../Cargo.toml"], False, {'CARGO_HOME': os.getcwd() + '/rust_crates'})
```

Comment:
Rather than using cwd, use the root_path defined above

Reply:
Oh didn't see that.